src/ssmDocument/commands/publishDocument.ts

```diff
@@ -27,15 +28,17 @@ export async function publishSSMDocument(awsContext: AwsContext, outputChannel:
 
     const textDocument = vscode.window.activeTextEditor?.document
     if (!textDocument) {
-        logger.error('Could not get active text editor for local SSM Document definition')
-        throw new Error('Could not get active text editor for local SSM Document definition')
+        let error = new Error('Could not get active text editor for local SSM Document definition')
+        logger.error(error.message)
+        throw error
     }
 
-    if (textDocument.languageId !== 'ssm-json' && textDocument.languageId !== 'ssm-yaml') {
-        logger.error(
-            'Could not get SSM Document from current active text editor. Please set the document language to ssm-yaml or ssm-json.'
+    if (textDocument.languageId !== ssmJson && textDocument.languageId !== ssmYaml) {
+        let error = new Error(
+            'Current editor language does not match the supported formats. Please set the document language to ssm-yaml or ssm-json.'
```

Comment:
Try this format:

Current editor language does not match the supported formats: X

Where X is an array of ssmJson, ssmYaml, string joined on ", "

Reply:
Will try this format